src/Symfony/Component/EventDispatcher/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * The signature of the `EventDispatcherInterface::dispatch()` method should be updated to `dispatch($event, string $eventName = null)`, not doing so is deprecated
+ * added the `EventListenerInterface` interface and guessed event name from the `__invoke` method signature.
```

Comment:
should we move it to contracts?

Reply:
This is an empty interface used as marker for CompilerPass? I'm not sure it's worth it

Reply:
Why not? I see the benefit on my side: allow using the interface without require the component. What's the downside?

Reply:
moved into Contract